src/common/docsHelpers.js

```diff
@@ -25,7 +25,8 @@ const getUserAttrs = (walkthroughId, username) => ({
     walkthroughId || buildValidProjectNamespaceName(username, 'shared')
   ),
   'user-username': username,
-  'user-sanitized-username': cleanUsername(username)
+  'user-sanitized-username': cleanUsername(username),
+  'walkthrough-locations': process.env.WALKTHROUGH_LOCATIONS
```

Comment:
@aidenkeating  this isn't working for me, any ideas?

Reply:
@finp I believe `docsHelpers.js` is only used from the client-side? The browser wouldn't have access to the env var by default. We could make a change to pass it from the server to the client if we need it.

Reply:
@pb82 seems like we need a new plan? But you mentioned that there's a plan to add default value to tutorial web app operator, right? That would also solve my issue in docs, if there's a jira?

Reply:
@finp ah right, sorry. I forgot that that code is only ever run on the client. Looking for that JIRA now.